doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -566,78 +566,96 @@ The query returns all the entities that match all the matching conditions (AND l
 
 There are two kinds of statements: *unary statements* and *binary statements*.
 
-Binary statements are composed by an attribute name (e.g. "temperature"), an operator and a value
+Binary statements are composed by an attribute path (e.g. "temperature"), an operator and a value
 (whose format depends on the operator), e.g.:
 
 ```
 temperature==50
 temperature<=20
 ```
 
+The attribute path is used to select the *element under evaluation* for the statement. Its general
```

Comment:
The *value under test* is defined as the value of the JSON property addressed by the list of tokens described above.